tests/karate/src/test/java/be/features/utils/constants.feature

```diff
@@ -12,3 +12,20 @@ Feature: Constants
 
     * def rootChannel = '/root'
     * def random = Java.type('be.utils.RandomUtils')
+
+    # Paths to util features
+    * def utilsPath = 'classpath:be/features/utils/'
+    * def serverFeature = utilsPath + 'server.feature'
+    * def mockClientFeature = utilsPath + 'mockClient.feature'
+
+    # Paths to scenarios defined in simpleScenarios
+    * def simpleScenario = 'classpath:be/features/utils/simpleScenarios.feature@name='
+    * def createLaoScenario = simpleScenario + 'valid_lao'
+    * def createRollCallScenario = simpleScenario + 'valid_roll_call'
```

Comment:
I find `create` and `valid` quite different and a bit confusing. Shouldn't the `valid_roll_call` be renamed `create_roll_call` to avoid this confusion ? Other proposition ?

Reply:
Good point, I forgot to change this.